style.css

```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+body {
+  background-color: #e5e5e5;
+  padding: 0;
+  margin: 0;
+}
+
+.intro {
+  background-image: url(./figmaImages/images/background-shapes.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  object-fit: contain;
+  max-width: fit-content;
+  width: 100%;
+  height: 718px;
+}
```

Comment:
- [ ] Good job so far 👍👍However, at `375px` the background image looks a bit different from what we have in the Figma template. Please make sure to fix this and use the correct image for getting a design close to the Figma template as much as possible according to the project requirements 

Your design | Figma template
---| ---|
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/106119105/2ef73632-ac86-4480-9d52-d86049d32788) | ![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/106119105/9ba31dd3-3457-43a3-9a9c-432bc266a35f)